pages/jams/[jam].js

```diff
@@ -175,32 +183,41 @@ const Jam = () => {
         description={jam && jam.description}
         participantId={participantId}
       />
+      {question && (
+        <Progress
+          h="0.5"
+          value={
+            (question.meta.numVotes / question.meta.numQuestions) *
+            100
+          }
+        />
+      )}
       <Layout py={14}>
         <GridItem colSpan={6}>
-          <Heading as="h1" size="xl" fontWeight={500} mb={8}>
-            {!isDone
-              ? question
-                ? question.text
-                : 'Loading...'
-              : 'All done'}
+          {question && (
+            <Text color="gray.400">
+              {question.meta.numVotes + 1} /{' '}
+              {question.meta.numQuestions}
+            </Text>
+          )}
+          <Heading as="h1" size="xl" mb={8}>
+            {!isDone ? question.text : 'All done'}
           </Heading>
         </GridItem>
-        {question
-          ? question.isUserSubmitted && (
-              <GridItem colSpan={4}>
-                <HStack>
-                  <InfoOutlineIcon />
-                  <Text fontSize="xs">
-                    <strong>Participant</strong> submitted{' '}
-                    {moment(
-                      question.createdAt?._seconds * 1000,
-                    ).format('DD MMM hh:mm A')}
-                  </Text>
-                </HStack>
-                <br />
-              </GridItem>
-            )
-          : ''}
+        {question && question.isUserSubmitted && (
+          <GridItem colSpan={4}>
+            <HStack>
+              <InfoOutlineIcon />
+              <Text fontSize="xs">
+                <strong>Participant</strong> submitted{' '}
+                {moment(question.createdAt?._seconds * 1000).format(
+                  'DD MMM hh:mm A',
+                )}
```

Comment:
You can use `convertDate` from `utils/date.js` to do this so we keep things consistent.

Reply:
Thanks will update